101-javaee-like-getting-started/pom.xml

```diff
@@ -47,10 +46,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-scheduler</artifactId>
         </dependency>
-        <!-- <dependency>  https://github.com/quarkusio/quarkus/issues/2725
-      <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-smallrye-jwt</artifactId>
```

Comment:
Commit is called `Add new dependency to 101 javaee getting-started project as issue was…
… resolved` but you are adding SR JWT. I don't understand it.

Reply:
I renamed commit to `Add quarkus-smallrye-jwt extension to 101 javaee getting-started project as issue was resolved`